recipes/_ghost.rb

```diff
@@ -4,6 +4,10 @@
     not_if { ::File.exist?("#{Chef::Config[:file_cache_path]}/ghost.zip") }
 end
 
+directory node['ghost-blog']['install_dir'] do
```

Comment:
Yeah, my problem was that unless you included the nginx recipe, the parent dir didn't exist, so the unzip would fail. 

So, I added this check and (in a subsequent commit on my new branch, admittedly), changed the conditional to check for the `index.js` file. 
`not_if { ::File.exists?("#{install_dir}/index.js") }`

Does that sound ok to you?